site/content/docs/aws-computes/app-runner.md

```diff
@@ -1 +1,8 @@
-### TODO
\ No newline at end of file
+# AWS App Runner - fully managed service
```

Comment:
Delete this whole folder for `aws-computes` since it is now covered in the `support.md` file.

Reply:
done